flujo/cli/main.py

```
@@ -1783,17 +1809,324 @@
+                from flujo.domain.pipeline_validation import (
+                    ValidationReport as _VR,
+                    ValidationFinding as _VF,
+                )
+
+                def _vf_list(arr: list[dict[str, Any]]) -> list[_VF]:
+                    out: list[_VF] = []
+                    for it in arr:
+                        try:
+                            out.append(_VF(**it))
+                        except Exception:
+                            continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)